lib/string_tools.rb

```diff
@@ -202,6 +211,14 @@ def normalize_link(node, attr_name)
       rescue Addressable::URI::InvalidURIError
         node.swap node.children
       end
+
+      def remove_links(node, attr_name)
+        return unless node[attr_name]
+
+        node[attr_name] = node[attr_name].gsub(URI::regexp, '').squish
```

Comment:
Style/ColonMethodCall: Do not use :: for method calls.<br>Lint/UriRegexp: URI.regexp is obsolete and should not be used. Instead, use URI::DEFAULT_PARSER.make_regexp.